tutorials/setup.md

```diff
@@ -39,6 +39,57 @@ yum install -y \
   runc
 ```
 
+RHEL 8 distributions:
+
+Make sure you are subscribed to the following repositories: \
+BaseOS/x86_64 \
+Appstream/x86_64 
+
+Follow the guide below to subscribe to the repositories if not already subscribed: \
+https://access.redhat.com/solutions/265523
+
+This required go version 12.1 or greater. Use the link below to install the appropriate go version for the system:\
+(Replace apt-get with yum for Linux based machines) \
```

Comment:
Might want to rephrase this to say RHEL machines instead of Linux based, since that is a very broad term